src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -109,6 +110,11 @@ public BinaryData getUnstructuredAudioBlob(UUID location) {
         return dataManagementApi.getBlobDataFromUnstructuredContainer(location);
     }
 
+    @Override
+    public BinaryData getDetsAudioBlob(UUID location) {
+        return dataManagementApi.getBlobDataFromDetsContainer(location);
```

Comment:
Call BlobContainerDownloadable.downloadBlobFromContainer to avoid bringing downloads into memory